src/lib/babylon/visual/ambientLights.ts

```diff
@@ -12,7 +12,7 @@ export function setupEnvironment(scene: BABYLON.Scene) {
   skyMaterial.luminance = 1;
   skyMaterial.turbidity = 5;
   skyMaterial.useSunPosition = true;
-  skyMaterial.sunPosition.set(-1, 0.81, -1).scaleInPlace(5000000)
+  skyMaterial.sunPosition.set(-1, 0.81, -0.75).scaleInPlace(5000000)
```

Comment:
_Doubt_
Using huge numbers to basically indicate infinity seems to be a common pattern. I wonder if this kind of method should actually support taking `Infinity` and choosing an adequate number by itself.

Reply:
we are multiplying matrices here, we certainly prefer a non-infinity number to prevent breaking the renderer.

are you suggesting it to prevent the arbitrary number?